sktime/transformations/series/boxcox.py

```diff
@@ -266,22 +279,18 @@ def _transform(self, X, y=None):
         """
         from scipy.special import boxcox
 
-        enforce_positive = self.enforce_positive
```

Comment:
could you leave these at the start?

The access of `self` is intentionally here at the start, so a reader of the code can see explicitly all the variables from `self` that are being accessed.